protocol/js/flow-types.js

```diff
@@ -307,20 +303,20 @@ export type config_Stream = {
   fd: int;
 }
 
-export type config_LogLevel = 'NONE_0' | 'DEBUG_1' | 'INFO_2' | 'NOTICE_3' | 'WARN_4' | 'ERROR_5' | 'CRITICAL_6' | 'FATAL_7'
+export type config_LogLevel = 0 /* 'NONE_0' */ | 1 /* 'DEBUG_1' */ | 2 /* 'INFO_2' */ | 3 /* 'NOTICE_3' */ | 4 /* 'WARN_4' */ | 5 /* 'ERROR_5' */ | 6 /* 'CRITICAL_6' */ | 7 /* 'FATAL_7' */
```

Comment:
we really get numbers, 0, 1, 2 ... from the service, but I originally wanted to keep the semantic meaning there. Now we have both!